charts/substrate-telemetry/values.yaml

```diff
@@ -32,6 +32,12 @@ envVars:
     # The frontend docker container makes this available to the UI,
     # so that it knows where to look for feed information:
     SUBSTRATE_TELEMETRY_URL: "wss://core-service-domain.com"
+    
+extraArgs:
+  shard: {}
```

Comment:
The variables `extraArgs.shard` and `extraArgs.frontend` have been defined but are not used in the template. I think you might have just forgotten to `git add` them.

Reply:
I added the usage in the shard template. Instead, I removed `extraArgs.frontend` since the frontend app does not expect any argument to be passed (it just uses environment variables and therefore rely on `envVars.fronted`).
wdyt @PierreBesson ?

Reply:
Perfect